src/ProjectService.Mappers/Db/DbProjectImageMapper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using LT.DigitalOffice.ProjectService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Mappers.Db
+{
+    public class DbProjectImageMapper : IDbProjectImageMapper
+    {
+        public DbProjectImage Map(CreateImageRequest request, Guid imageId)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            return new DbProjectImage
+            {
+                Id = Guid.NewGuid(),
+                ImageId = imageId,
+                ProjectId = request.ProjectOrTaskId
```

Comment:
its strange, why projectId can contains projectId and TaskTd?

Reply:
We just have one model to get an image of the project and task. Therefore, this field name was chosen.